plugins/base/src/main/resources/dokka/scripts/platformContentHandler.js

```diff
@@ -42,15 +42,14 @@ function handleAnchor() {
         if (element) {
             let tab = searchForTab(element)
             if (tab) {
-                let found = document.querySelector('.tabs-section > .section-tab[data-togglable="' + tab.getAttribute("data-togglable") + '"]')
                 toggleSections(tab)
-                const content = element.nextElementSibling
-                if(content){
-                    content.classList.add('anchor-highlight')
-                    highlightedAnchor = content
-                }
-                element.scrollIntoView({behavior: "smooth"})
             }
+            const content = element.nextElementSibling
```

Comment:
This change is unrelated to anchors rendering, but I saw that anchors are not highlighted correctly when there is no tab (eg. on module page)